packages/screens/README.md

```diff
@@ -0,0 +1,54 @@
+# `@limbo-works/screens`
+
+> A tool for parsing TailwindCSS `screens` from the standard Limbo breakpoint
+> format (as well as a helpful `breakpoints` object)
+
+## Installation
+
+Before installing the package, it's necessary to configure NPM with a GitHub
+[Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
+(PAT) with the `read:packages` scope.
+
+Using your PAT, configure the `.npmrc` in the root of the project like so
+_before_ running the install command:-
+
+```npmrc
+//npm.pkg.github.com/:_authToken=${PAT}
+@limbo-works:registry=https://npm.pkg.github.com
+```
+
+Install the package:-
+
+```shell
+$ yarn add @limbo-works/screens
+```
+
+## Usage
+
+```js
+// ~/frontend/assets/js/screens.js
+
+const { default: configure } = require('@limbo-works/screens');
+
+const { breakpoints, screens } = configure([
```

Comment:
breakpoints and screens can be a bit confusing for people not familiar with tailwind screens. This can be alright as long as the docs show an example. (minor)

Reply:
Wrote a comment on #1, but much to the same point - maybe naming of properties should be more explicit, like `twScreens` or something similar as suggested on the issue?

Don't know if `breakpoints` should be renamed to `breakpointData` or `rawBreakpointData` or something like that as well, or if `breakpoints` covers it well enough (very minor). But the tailwind specification would at the very least be nice.

But yeah, a lot can be explained through documentation as well. Though it will at to code readability, which is nice.